.circleci/config.yml

```diff
@@ -486,8 +450,5 @@ workflows:
   Release Build:
     when: << pipeline.parameters.release_build >>
     jobs:
-      - lint
-      - Release Build:
-          requires:
-            - lint
+      - Release Build
```

Comment:
We'll need to remember to duplicate this dependency when migrating the release build job to Buildkite 👍

Reply:
Yeap. I think we should add all checks (lint, detekt, test) before we build the release artifacts 🤷

Reply:
Great idea!

Reply:
I actually wanted to mention it for iOS because I noticed that the release build doesn't have tests as a gate. While unlikely, that leaves the door open to a build with failing tests being deployed to testers, and in worse case scenario real users.